Forget/Forget.css

```diff
@@ -0,0 +1,35 @@
+.forget__box{
+    position: relative;
```

Comment:
You can use some code formatter like prettier, for more readable code.

Reply:
okay i will look at it..Thanks